src/pages/LoginPage.js

```diff
@@ -1,23 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState} from 'react';
 import image from '../images/image-1.png';
 import { useNavigate} from 'react-router-dom';
 
 
 function Login() {
     const [Username, setUsername] = useState('');
     const [Password, setPassword] = useState('');
-    const navigate = useNavigate();;
-
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
We need to control if `username` exists in local storage in useEffect.
We can delete this `isLoggedIn` part because if the user already logged in, we need to navigate the user directly to tasks page.